nats.go

```diff
@@ -2683,8 +2687,19 @@ func (nc *Conn) processMsg(data []byte) {
 				sub.pTail = m
 			}
 		}
-		if hasFC {
-			jsi.trackSequences(m)
+		if jsi != nil && (hasHBs || hasFC) {
```

Comment:
I thought I needed these tracked before if hasFC but the way the impl worked we do not need to track if hasFC per se.

Reply:
Should this block get moved to another location?

Reply:
I think this is only applicable for HBs now.

Reply:
Removed the `hasFC` check.
```diff
-if jsi != nil && (hasHBs || hasFC) {
+if jsi != nil && hasHBs {
```